Curtains/Curtains.Domain/Data/CurtainsContext.cs

```diff
@@ -0,0 +1,351 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Curtains.Infrastructure.Models
+{
+    public partial class CurtainsContext : DbContext
+    {
+        public CurtainsContext()
+        {
+        }
+
+        public CurtainsContext(DbContextOptions<CurtainsContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Bracing> Bracings { get; set; } = null!;
+        public virtual DbSet<Color> Colors { get; set; } = null!;
+        public virtual DbSet<Curtain> Curtains { get; set; } = null!;
+        public virtual DbSet<Decoration> Decorations { get; set; } = null!;
+        public virtual DbSet<DeliveryType> DeliveryTypes { get; set; } = null!;
+        public virtual DbSet<Fabric> Fabrics { get; set; } = null!;
+        public virtual DbSet<Manufacturer> Manufacturers { get; set; } = null!;
+        public virtual DbSet<MarketingInfo> MarketingInfos { get; set; } = null!;
+        public virtual DbSet<Material> Materials { get; set; } = null!;
+        public virtual DbSet<Model> Models { get; set; } = null!;
+        public virtual DbSet<Order> Orders { get; set; } = null!;
+        public virtual DbSet<OrderCurtain> OrderCurtains { get; set; } = null!;
+        public virtual DbSet<Role> Roles { get; set; } = null!;
+        public virtual DbSet<User> Users { get; set; } = null!;
+        public virtual DbSet<UserOrder> UserOrders { get; set; } = null!;
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. You can avoid scaffolding the connection string by using the Name= syntax to read it from configuration - see https://go.microsoft.com/fwlink/?linkid=2131148. For more guidance on storing connection strings, see http://go.microsoft.com/fwlink/?LinkId=723263.
+                optionsBuilder.UseSqlServer("Server=(LocalDB)\\MSSQLLocalDB;Database=Curtains;Trusted_Connection=true;");
+            }
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.UseCollation("Cyrillic_General_CI_AS");
+
+            modelBuilder.Entity<Bracing>(entity =>
```

Comment:
Are you sure that we need to describe every entity in this way? If we will have 100 entities => we will have file with 2000 lines of code. Looks like it's worst way,isn't it?
Better use ` modelBuilder.ApplyConfiguration()`
This [article](https://digitaldrummerj.me/ef-core-split-model-config/) can help you with it.